components/schemes/Color/tests/unit/MDCSemanticColorSchemeTests.m

```diff
@@ -177,4 +177,23 @@ - (BOOL)compareColorsWithFloatPrecisionFirstColor:(UIColor *)firstColor
           MDCCGFloatEqual(fAlpha, sAlpha));
 }
 
+- (void)testColorSchemeCopyTest {
+  // Given
+  MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] init];
+  MDCSemanticColorScheme *colorSchemeCopy = [colorScheme copy];
+
+  // Then
+  XCTAssertFalse(colorScheme == colorSchemeCopy);
+  XCTAssertEqual(colorScheme.primaryColor, colorSchemeCopy.primaryColor);
```

Comment:
Please use XCTAssertEqualObjects. Since each of these properties has `copy` semantics, we should allow copied colors and not fail if they are not the same pointer.

See https://nshipster.com/equality/ for a good article on equality vs. identity.

Reply:
Addressed.